.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -56,8 +56,8 @@ body:
     attributes:
       label: Environment Details
       placeholder: |
-        - Device: [e.g. iPhone 15 Pro, iPad Air (5th Generation)]
-        - OS Version: [e.g. 17.4.1]
+        - Device: [e.g. iPhone 16 Pro, iPad Air (5th Generation)]
+        - OS Version: [e.g. 18.0.1]
```

Comment:
⛏️ This should be `18.1` as far as I can see in `.test-simulator-ios-version`

Reply:
I intentionally made it different, because it's giving it as an example

Reply:
Ohhh makes sense!